### PR TITLE
Add the parser to the runner

### PR DESCRIPTION
**Pull request by Dexterp37**

### CODE REVIEW

File: parsers/histogram_tools.py [hunk 1]
```diff
@@ -408,8 +408,11 @@ def linear_bucket_parameters(definition):
 
     @staticmethod
     def enumerated_bucket_parameters(definition):
-        n_values = definition['n_values']
-        return (1, n_values, n_values + 1)
+        try:
+            n_values = definition['n_values']
+            return (1, n_values, n_values + 1)
+        except:
+            return (1, 2, 2 + 1) # TODO totally random.
```
Comment (georgf) [inline parsers/histogram_tools.py]: # TODO: Fixup old non-number values & expressions.
    # History: bug 920169, bug 1245910
    # "JS::gcreason::NUM_TELEMETRY_REASONS"
    # "JS::gcreason::NUM_TELEMETRY_REASONS+1"
    # "mozilla::StartupTimeline::MAX_EVENT_ID"

File: parsers/histogram_tools.py [hunk 2]
```diff
@@ -242,7 +242,7 @@ def verify_attributes(self, name, definition):
                        lambda allowed_keys: Histogram.check_keys(name, definition, allowed_keys))
 
         self.check_name(name)
-        self.check_field_types(name, definition)
+        # self.check_field_types(name, definition)
```
Comment (georgf) [inline parsers/histogram_tools.py]: Can we use the strict_type_check flag for skipping these things?

File: parsers/histogram_tools.py [hunk 3]
```diff
@@ -408,8 +409,16 @@ def linear_bucket_parameters(definition):
 
     @staticmethod
     def enumerated_bucket_parameters(definition):
-        n_values = definition['n_values']
-        return (1, n_values, n_values + 1)
+        try:
+            n_values = definition['n_values']
+            return (1, n_values, n_values + 1)
+        except:
+            # TODO: Fixup old non-number values & expressions.
```
Comment (georgf) [inline parsers/histogram_tools.py]: Can you open an issue on that?

Comment (Dexterp37) [inline parsers/histogram_tools.py]: Opened issue #6.

**Pull request merged**